scout/constants/indexes.py

```diff
@@ -30,21 +30,12 @@
         ),
     ],
     "variant": [
-        IndexModel(
-            [
-                ("case_id", ASCENDING),
-                ("category", ASCENDING),
-                ("variant_rank", ASCENDING),
-            ],
-            name="caseid_variantrank",
-            background=True,
-        ),
         IndexModel(
             [
                 ("case_id", ASCENDING),
                 ("category", ASCENDING),
                 ("variant_type", ASCENDING),
-                ("rank_score", DESCENDING),
+                ("variant_rank", ASCENDING),
```

Comment:
Quite agree on the removal, but I think we can be more efficient with the addition, squeezing that hgnc_ids-as-a-range-level-field fix as well. If its ok I'll make a suggestion?

Reply:
> Quite agree on the removal, but I think we can be more efficient with the addition, squeezing that hgnc_ids-as-a-range-level-field fix as well. If its ok I'll make a suggestion?

I wanted to add it as well, but was unsure if we wanted in another PR. But sure, add it to this!